server/gitrest/packages/gitrest-base/src/utils/redisFs/redisFsManager.ts

```diff
@@ -291,7 +301,7 @@ export class RedisFs implements IFileSystemPromises {
 	public async stat(filepath: PathLike, options?: any): Promise<Stats | BigIntStats> {
 		const filepathString = filepath.toString();
 		const data = await executeRedisFsApi(
-			async () => this.redisFs.get<Buffer | string>(filepathString),
+			async () => this.redisFsClient.get<string | Buffer>(filepathString),
```

Comment:
Now that mkdir uses a "Folder" dummy value when creating directories, shouldn't we use that value to pass to the `getStats()` function below? Otherwise, how can it tell folders apart from files?